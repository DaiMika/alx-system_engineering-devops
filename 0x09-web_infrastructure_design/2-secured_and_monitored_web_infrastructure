https://github.com/DaiMika/alx-system_engineering-devops/blob/master/0x09-web_infrastructure_design/2-secured_and_monitored_web_infrastructure.draft

https://github.com/DaiMika/alx-system_engineering-devops/blob/master/0x09-web_infrastructure_design/files.md/2-secured_and_monitored_web_infrastructure.md

https://github.com/DaiMika/alx-system_engineering-devops/blob/master/0x09-web_infrastructure_design/snap/2-secured_and_monitored_web_infrastructure.snap
