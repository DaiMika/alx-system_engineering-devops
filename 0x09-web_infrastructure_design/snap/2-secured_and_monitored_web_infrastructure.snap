https://docs.google.com/document/d/1qHipePavgSB4NwDgFwHxZhZdk8FFgyYVyXXe6A16-QE/edit?usp=drivesdk
